pre-index.php

```diff
@@ -426,6 +426,23 @@ class="onoffswitch-checkbox" checked>
                             </select>
                         </div>
                     </div>
+                    <div id="gyms-raid-filter-wrapper" style="display:none">
+                        <?php if ($map === "monocle" && !$noExEligible) {
```

Comment:
Wouldn't this be better? `$fork ==="alternate"`

Reply:
Yes I'll change and update the pr